ncm-dpmlfc/src/main/pan/components/dpmlfc/schema.pan

```diff
@@ -8,6 +8,14 @@ declaration template components/dpmlfc/schema;
 
 include { 'quattor/schema' };
 
+function component_dpmlfc_number_string_valid = {
+  if ( !match(ARGV[0],'^\d+$') ) {
+    error(format("String '%s' is not a valid number",ARGV[0]));
+    return(false);
```

Comment:
I know but I always add it to make it clear to people less aware about pan validation functions... Is it a problem (I have it in many validation functions I wrote in various config modules!)?